memoize/wrapper.py

```diff
@@ -96,6 +96,9 @@ async def refresh(actual_entry: Optional[CacheEntry], key: CacheKey,
         except Exception as e:
             logger.debug('Early failure instantiating coroutine for %s: %s', key, e)
             raise CachedMethodFailedException('Refresh failed to start') from e
+        except BaseException as be:
+            logger.debug("Non-standard exception caught during coroutine instantiation for %s: %s", key, be)
+            raise CachedMethodFailedException('Refresh failed to start') from be
```

Comment:
I'd probably _only_ catch BaseException here -- an invocation-time vs execution-time failure is a corner case to begin with.

Reply:
Done, updated the areas where we _catch or check for instanceof_ `Exception` to `BaseException`. That way we're being comprehensive.